### PR TITLE
[Chat] Expose send_message API from backend. 

### PR DESCRIPTION
**Pull request by tangziyi001**
The send message API can:
1. Store the message as a user action
2. Send to pusher so that the client can get the message in real-time

Unit test + real request:
screen shot 2019-02-20 at 9 31 15 pm

### CODE REVIEW

File: new_dawn_server/pusher/chat_service.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+USER_FROM_ID = "user_from_id"
+USER_TO_ID = "user_to_id"
+
+class ChatService:
+
+	def __init__(self, user_me_id=None, user_you_id=None):
+		self.user_me_id = user_me_id
+		self.user_you_id = user_you_id
+
+	def _build_channel_id(self):
```
Comment (duckmoll) [inline new_dawn_server/pusher/chat_service.py]: Can we make the channel id ignorant of numerical meaning?

File: new_dawn_server/settings.py [hunk 2]
```diff
@@ -140,3 +140,9 @@
 
 # Authy Application Key
 ACCOUNT_SECURITY_API_KEY = os.environ.get('ACCOUNT_SECURITY_API_KEY')
+
+# Pusher Application Key
+PUSHER_APP_KEY=os.environ.get('PUSHER_APP_KEY')
```
Comment (duckmoll) [inline new_dawn_server/settings.py]: Can you share this? Thanks!

File: new_dawn_server/pusher/chat_service.py (hunk 1 above)
Comment (tangziyi001) [reply new_dawn_server/pusher/chat_service.py]: What do you mean by numerical meaning exactly? This is to make sure that each pair of users have an unique channel ID. In the future I will use a hash of this as channel ID.

File: new_dawn_server/settings.py (hunk 2 above)
Comment (tangziyi001) [reply new_dawn_server/settings.py]: Will create a google drive folder for these credentials.

File: new_dawn_server/pusher/chat_service.py (hunk 1 above)
Comment (duckmoll) [reply new_dawn_server/pusher/chat_service.py]: I see. I meant the greater than relationship, but I think hash will be good.

**Pull request merged**